ddtrace/telemetry/__init__.py

```diff
@@ -0,0 +1,47 @@
+import six
+
+import ddtrace.internal.telemetry.telemetry_writer
+
+
+class _InstrumentationTelemetryStatus(type):
+    @property
+    def _enabled(_):
+        # type: () -> bool
+        """Instrumentation Telemetry enabled status."""
+        return ddtrace.internal.telemetry.telemetry_writer.TelemetryWriter.enabled
+
+
+class InstrumentationTelemetry(six.with_metaclass(_InstrumentationTelemetryStatus)):
```

Comment:
Why "InstrumentationTelemetry"? This just uses `TelemetryWriter` which does more than just instrumentation telemetry